src/js/utils/specs/date-utils-spec.js

```diff
@@ -0,0 +1,145 @@
+define(
+    [
+        '../../utils/spec-helpers/spec-helpers.js',
+        '../date-utils.js',
+        'moment',
+        'moment-timezone'
+    ],
+    function(
+        SpecHelpers,
+        DateUtils,
+        moment,
+        momentTZ
+    ) {
+        'use strict';
+
+        describe('DateUtils', function() {
+            beforeEach(function() {
+                setFixtures('<div class="test"></div>');
+            });
+
+            describe('stringToMoment', function() {
+                it('returns an empty string if provided no time', function() {
+                    expect(DateUtils.stringToMoment()).toEqual('');
+                });
+                it('will convert a string to an moment.js object', function() {
+                    expect(
+                        DateUtils.stringToMoment('2016-10-14 08:00:00+00:00').format('ll')
+                    ).toEqual('Oct 14, 2016');
+                });
+            });
+
+            describe('getSysTimezone', function() {
+                it('attempts to determine users timezone', function() {
```

Comment:
Nit: not necessarily "users", but the "system".  This general purpose toolkit could be used even server-side (in node.js, for example).